File: 01.ProgrammingFundamentals/Java/01.IntroAndBasicSyntax/Exercises/src/P05_CharacterStats.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class P05_CharacterStats {
    public static void main(String[] args) throws IOException {

        BufferedReader console = new BufferedReader(new InputStreamReader(System.in));
        String name = console.readLine();
        int currentHealth = Integer.parseInt(console.readLine());
        int maximumHealth = Integer.parseInt(console.readLine());
        int currentEnergy = Integer.parseInt(console.readLine());
        int maximumEnergy = Integer.parseInt(console.readLine());

        System.out.format("Name: %s" + System.lineSeparator() +
                          "Health: |%s|" + System.lineSeparator() +
                          "Energy: |%s|",
                          name,
                          Health(currentHealth,maximumHealth),
                          Health(currentEnergy, maximumEnergy));
    }
    private static String Health(int current, int max) {
        String rem = "";
        for(int i = 1; i <= current; i++) {
            rem += "|";
        }
        for(int j = 1; j <= max - current; j++) {
            rem += ".";
        }
        return rem;
    }
}

File: ProgrammingFundamentals/C#/01.IntroAndBasicSyntax/Exercises/P05_CharacterStats/P05_CharacterStats.cs
using System;

namespace P05_CharacterStats
{
    public class P05_CharacterStats
    {
        static void Main(string[] args)
        {
            string name = Console.ReadLine();
            int currentHealth = int.Parse(Console.ReadLine());
            int maximumHealth = int.Parse(Console.ReadLine());
            int currentEnergy = int.Parse(Console.ReadLine());
            int maximumEnergy = int.Parse(Console.ReadLine());

            Console.WriteLine($"Name: {name}");
            Console.WriteLine($"Health: |{new string('|', currentHealth)}" +
                              $"{new string('.', maximumHealth - currentHealth)}|");

            Console.WriteLine($"Energy: |{new string('|', currentEnergy)}" +
                              $"{new string('.', maximumEnergy - currentEnergy)}|");
        }
    }
}